spec/Classes.tex

```diff
@@ -58,7 +59,8 @@ \section{Class Declarations}
 Inheritance is described in~\rsec{Inheritance}.
 
 \begin{future}
```

Comment:
I do not like how this future is written altogether. Also, what it describes can hardly be called "inheritance".

I would rather it be called "Open Issue" and contain just one sentence like "Classes that inherit from records are an area for future work."